### ISSUE TITLE
Duplicate posts but do not have Sociable or SexyBookmarks installed [moved]

### ISSUE THREAD

**Issue by zyml** [zyml/arras-theme#46]
This is [Issue 130](http://code.google.com/p/arras-theme/issues/detail?id=130) moved from a Google Code project.
Added by 2011-04-22T14:18:29.000Z by [ad...@graphicdistortions.com](http://code.google.com/u/108840122228546730547/).
Please review that bug for more context and additional comments, but update this bug.

Original labels: Type-Defect, Priority-Medium
### Original description

```
<b>What steps will reproduce the problem?</b>
Have a total of 11 posts
     {5 sticky in slideshow, 1 private, 5 normal posts }

Now I have NO TITLE defined in the News Posts and setup as 4 in the Featured/4 in the News

The 5 sticky show in the slideshow fine.... {it's defined for 5}
The 4 in the featured posts fine
2 in the News section...As it's showing a link to the private but that's fine, and ONLY the 2 since that's now everything
...
And I have a link to Older Posts   (or Page 2 if the PageNavi is enabled, but it was there before I installed it)
...
in short.... What the heck ?
...
Page 2 is showing one of the stickied posts already shown in the slideshow.

<b>What is the expected output? What do you see instead?</b>
there should be no page2 generated

<b>What versions of WordPress and Arras Theme are you using?</b>
Latest version wordpress (3.1.1) Arras 1.5.0.1

<b>Please include the link to your installation.</b>
screenshots attached in zip, but WILL provide an admin link as the site is non-live
```
